sdk/Lusid.Sdk.Tests/Utilities/ApiRetryHandlerTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Utilities;
+using NUnit.Framework;
+using Polly;
+using RestSharp;
+
+namespace Lusid.Sdk.Tests.Utilities
+{
+    [TestFixture]
+    public class ApiRetryHandlerTests
+    {
+        private LusidApiFactory _apiFactory;
+        private readonly Policy<IRestResponse> _initialRetryPolicy = RetryConfiguration.RetryPolicy;
+        private HttpListener _httpListener;
+        private const string ListenerUriPrefix = "http://localhost:4444/";
+        
+        [SetUp]
+        public void SetUp()
+        {
+            _httpListener = new HttpListener();
+            _httpListener.Prefixes.Add(ListenerUriPrefix);
+            
+            var testApiConfig = TestLusidApiFactoryBuilder.CreateApiConfiguration(null);
+            testApiConfig.ApiUrl = ListenerUriPrefix;
+            
+            _apiFactory = new LusidApiFactory(testApiConfig);
+        }
+
+        [Test]
+        [Explicit("This passes locally, but not in the pipeline, and no idea why. Setting to explicit until resolved.")]
```

Comment:
Try running this test inside a docker container on your local machine, that should give you insight into what this isn't passing in the pipeline but is locally. We can't merge this until resolved and the test is in place.

Reply:
Good shout on using docker. I'll have a go